Taskfile.yml

```diff
@@ -42,11 +42,16 @@ tasks:
 
   kind_create:
     cmds:
-      - ./k8s/kind/kind-cluster.sh create
+      - '{{if eq .GOOS "windows"}}powershell -NoProfile -ExecutionPolicy Bypass -File ./k8s/kind/kind-cluster.ps1 create{{else}}./k8s/kind/kind-cluster.sh create{{end}}'
```

Comment:
The if conditions is lengthy and hurts readability.
Can we use Taskfile https://taskfile.dev/docs/guide#platform-specific-tasks-and-commands feature to define tasks for windows version separately?